tests/libstorage/storageclass.go

```diff
@@ -121,17 +118,27 @@ func CheckNoProvisionerStorageClassPVs(storageClassName string, numExpectedPVs i
 	// Verify we have at least `numExpectedPVs` available file system PVs
 	pvList, err := virtClient.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
 	Expect(err).ToNot(HaveOccurred())
+
+	if countNotLocalStoragePV(pvList, storageClassName) < numExpectedPVs {
+		Skip("Not enough available filesystem local storage PVs available, expected: %d", numExpectedPVs)
+	}
+}
+
+func countNotLocalStoragePV(pvList *k8sv1.PersistentVolumeList, storageClassName string) int {
```

Comment:
This actually counts local Storage as you have double negation: !notLocal

To make it easier to read change notLocalStoragePV function to isLocalStoragePV (just negate the return value), then everything should be simpler!

Reply:
the function name is still wrong, it counts local storage PVs, 

s/countNotLocalStoragePV/countLocalStoragePV/